server/matchmaker/matchmaker_queue.py

```diff
@@ -139,13 +139,13 @@ async def search(self, search: Search) -> None:
             # If the queue was cancelled, or some other error occurred,
             # make sure to clean up.
             self.game_service.mark_dirty(self)
-            if search in self.queue:
-                del self.queue[search]
+            if search in self._queue:
+                del self._queue[search]
 
     async def find_matches(self) -> None:
         self._logger.info("Searching for matches: %s", self.name)
 
-        if len(self.queue) < 2 * self.team_size:
+        if len(self._queue) < 2 * self.team_size:
```

Comment:
This assumes it will always be 2 teams. Should it be flexible to allow for 2v2v2 or 3v3v3v3?